refaster-support/src/main/java/tech/picnic/errorprone/refaster/matchers/IsEmpty.java

```diff
@@ -149,4 +234,9 @@ private static boolean isEmptyArrayCreation(ExpressionTree tree) {
             && ZERO.equals(ASTHelpers.constValue(newArray.getDimensions().get(0), Integer.class)))
         || (newArray.getInitializers() != null && newArray.getInitializers().isEmpty());
   }
+
+  private static Matcher<MethodInvocationTree> symbolReturns(TypePredicate predicate) {
+    return symbolMatcher(
+        (symbol, state) -> predicate.apply(((MethodSymbol) symbol).getReturnType(), state));
```

Comment:
Pitest flags that this lambda expression can be replaced with `null`. That's not correct; it's just confused because this code is only executed by a static initializer.